Update nf_research_tools.rdb.model.csv

allaway opened the pull request:
Follow up from our stand-up conversation today. 
The "disease" attributes have been changed to Genetic Disease, to more explicitly define what we are looking to describe here. 

Question for reviewers:
- does "genetic disease" sufficiently describe this
- We also have "manifestation" which can be used to capture the type of tumor. However, it's also to capture other terms that are not tumors such as behavioral or developmental traits. Should we duplicate the tumor information in another attribute (e.g. `tumorType`) to specifically capture this info and to make it easier to filter on this in the database?

anngvu commented:
Well, it depends on what ontology we want to use as the reference and be most consistent with. MONDO's branch of the matching concept is [hereditary disease](http://purl.obolibrary.org/obo/MONDO_0003847) and would include 10K+ terms. NCIT's exact match concept is [genetic disorder](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C3101?lang=en) and would include 256 terms. By using `genetic disease` instead of `disease`, we _are_ making things clearer/neater, but also imposing a restriction. So if we ever want to use the value "melanoma", that would be a little problematic because it is not considered a hereditary/genetic disease -- using it would be considered inconsistent with these other ontologies, e.g. [under MONDO](https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMONDO_0005105) it falls under several other parent classifications, but not `genetic disease`.

For now, I don't think the tools db should factor out the tumor types-specific manifestations from `manifestation` yet... The better way is to allow people to filter _upon_ the manifestation, not to have another column. It's like when I go to REI and 

For the workflow issue, I resolved this several months ago here: https://github.com/nf-osi/nf-metadata-dictionary/issues/217

allaway commented:
>For the workflow issue, I resolved this several months ago here: https://github.com/nf-osi/nf-metadata-dictionary/issues/217

Thanks! Yes, I copied your changes in from there :) 


>Well, it depends on what ontology we want to use as the reference and be most consistent with. MONDO's branch of the matching concept is [hereditary disease](http://purl.obolibrary.org/obo/MONDO_0003847) and would include 10K+ terms. NCIT's exact match concept is [genetic disorder](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C3101?lang=en) and would include 256 terms. By using genetic disease instead of disease, we are making things clearer/neater, but also imposing a restriction. So if we ever want to use the value "melanoma", that would be a little problematic because it is not considered a hereditary/genetic disease -- using it would be considered inconsistent with these other ontologies, e.g. [under MONDO](https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMONDO_0005105) it falls under several other parent classifications, but not genetic disease.

Thanks for these thoughts - To clarify, I _want_ to impose the restriction you described, so that we are not capturing all diagnoses (e.g. the cancers an NF patient is diagnosed with, or the cancers that an animal model models), and only the genetic diseases. In your example, the Genetic Disease/Disorder would be "not applicable" and the Manifestation (or tumor type, if we added that) would be "melanoma".  I am happy to use Genetic Disorder instead to describe this - for some reason that didn't pop up in OLS when I searched NCIT there.

Alternatively, we could _add_ Disease and Genetic Disorder, and then in your example it would be "melanoma" and "Not Applicable" respectively. But then also manifestation would be "melanoma", I suppose.

After thinking about it more I do want to avoid the term "hereditary" because NF1 can happen de novo (this is more relevant to biospecimen metadata though, less so for these research tools). 

>For now, I don't think the tools db should factor out the tumor types-specific manifestations from manifestation yet... The better way is to allow people to filter upon the manifestation, not to have another column.

I agree, good point. thanks!

allaway commented:
Updated to NCIT definition/term. I think we should also split this out to a normalized table (I think it is confusing to have "animal model genetic disorder" and "cell line genetic disorder" when we could just have "genetic disorder" in a separate normalized table").

cconrad8 commented:
@allaway Sorry late to the game - but I do want to add I have a bit of an issue with the definitions because they seem kind of broad - so what would someone put for the RAS mutation cell lines? (Which I believe rancho was going to add and what stemmed the issue if I remember correctly?

allaway commented:
Hey @cconrad8, 

If someone was to put a RAS mutation cell line, eg, KRAS G12V lung adenocarcinoma cell line, the Genetic Disorder would be Not Applicable, and the Manifestation would be Lung Adenocarcinoma.

cconrad8 commented:
@allaway Ok thanks, makes sense to me!
